Python/bytecodes.c

```diff
@@ -1,3 +1,11 @@
+// This file contains instruction definitions.
+// It is read by Tools/cases_generator/generate_cases.py
+// to generate Python/generated_cases.c.h.
+// Note that there is some dummy C code at the top and bottom of the file
+// to fool text editors like VS Code into believing this is valid C code.
+// The actual instruction definitions start at // BEGIN BYTECODES //.
```

Comment:
Nitpick: It's really the top and *middle* of this file. Might be worth mentioning the `// Families go below this point //` section, too.

Reply:
I'd like to leave finding that as an exercise for the reader, since that is work in progress. The generator currently doesn't actually see the families because it stops reading after `// END BYTECODES //` -- I was planning to fix that once families are actually used for anything.